salt/client/__init__.py

```diff
@@ -322,7 +322,7 @@ def run_job(
             >>> local.run_job('*', 'test.sleep', [300])
             {'jid': '20131219215650131543', 'minions': ['jerry']}
         '''
-        arg = salt.utils.args.condition_input(arg, kwarg)
+        arg = salt.utils.args.parse_input(arg, kwarg)
```

Comment:
parse_input takes `args` as one argument, this is passing kwarg to the `conditional` argument of parse_input

Reply:
i've updated it to a keyword arg